libs/model/src/models/thread.ts

```diff
@@ -164,3 +169,20 @@ export default (
       },
     },
   );
+
+export function getThreadSearchVector(title: string, body: string) {
```

Comment:
just a small detail, but maybe we should consider making this vector builder more generic? in case we need to support search on other types of content with title, body, etc

Reply:
This manual method of setting the search vector should only be used when the content to search over is not stored in Postgres. Given that we are unlikely to move any other content to S3/R2 (other than thread/comment bodies) in the near future I don't think this is worth the time atm.